### ISSUE TITLE
Error: No submodule mapping found in .gitmodules for path 'bson/specifications'

### ISSUE THREAD

**Issue by brianpowell** [go-mgo/mgo#307]
```
# cd /go/src/gopkg.in/mgo.v2; git submodule update --init --recursive
No submodule mapping found in .gitmodules for path 'bson/specifications'
```

**Comment by janajri** [comment]
I'm also seeing this currently. All our servers are unable to build.

**Comment by brianpowell** [comment]
We are not able to build either.  Looks like a symlink snuck into the `v2` 
<img width="983" alt="screen shot 2016-08-01 at 4 16 43 pm" src="https://cloud.githubusercontent.com/assets/2650513/17307561/b92f1bfe-5803-11e6-9158-8a216cf87060.png">

**Comment by niemeyer** [comment]
Having a look.. sorry about that.

**Comment by niemeyer** [comment]
That should have been fixed now in 0108465.

I don't know why tests didn't catch it.. the full grid was green:

https://travis-ci.org/go-mgo/mgo/builds/148986786

![image](https://cloud.githubusercontent.com/assets/378683/17309919/8528d49c-5841-11e6-92f4-647719c78e5e.png)

**Comment by niemeyer** [comment]
Ah, it probably breaks only if you're using submodules. Okay.

Tentatively closing it. Please drop me a note if you see any other issues.
